ci/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Install Miniconda
+if [ $(uname) == Linux ]; then
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+else
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh
+fi
```

Comment:
Let's remove this `if` and only install Miniconda3 for Linux (Circle doesn't support macOS on its free version),